docs/source/transformers/unsafe-options.rst

```diff
@@ -1,5 +1,5 @@
-Unsafe options transformation
-=============================
+Unsafe options ``(deprecated)``
+===============================
 
 .. warning::
```

Comment:
We might want to add a mention that this flag is scheduled for remove in 0.8.0 and migration to partial transformers is recommended (and users that want to watch their world burn can implement `[A] => Transformer[Option[A], A]`).

Reply:
I don't want to make any promises when this is going to be implemented.

And below there is already:

> Consider using :ref:`partial-transformers` instead.